test/ui-test/testSuites/global_shared/scripts/settings_names.py

```diff
@@ -150,6 +151,7 @@ class SettingsSubsection(Enum):
 backup_seed_phrase_popup_BackupSeedStepBase_confirmSecondWord_inputText = {"container": backup_seed_phrase_popup_BackupSeedStepBase_confirmSecondWord, "objectName": "BackupSeedStepBase_inputText", "type": "TextEdit", "visible": True}
 backup_seed_phrase_popup_ConfirmStoringSeedPhrasePanel_storeCheck = {"container": statusDesktop_mainWindow_overlay, "objectName": "ConfirmStoringSeedPhrasePanel_storeCheck", "type": "StatusCheckBox", "checkable": True, "visible": True}
 backup_seed_phrase_popup_BackupSeedModal_completeAndDeleteSeedPhraseButton = {"container": statusDesktop_mainWindow_overlay, "objectName": "BackupSeedModal_completeAndDeleteSeedPhraseButton", "type": "StatusButton", "visible": True}
+backup_seed_phrase_popup_step_StyledText = {"container": statusDesktop_mainWindow_overlay, "text": RegularExpression("Step*"), "type": "StyledText", "unnamed": 1, "visible": True}
```

Comment:
We are trying to avoid using `text` property in the object definition since it can change, for example, if we test the application in different language. 
My suggestion here is to add an object name in the `StyledText` object and updated it here accordingly.
The text can be verified in the assertion added in `SettingsScreen` but just calling the `object.text` property instead.

Reply:
Fixed